tests/query/processors/test_events_column_processor.py

```diff
@@ -30,7 +30,7 @@ def test_events_column_format_expressions() -> None:
             FunctionCall(
                 "the_message",
                 "coalesce",
-                (Column(None, None, "message"), Column(None, None, "search_message"),),
+                (Column(None, None, "search_message"), Column(None, None, "message")),
```

Comment:
It's so weird that we do this, since "search_message" is directly derived from message. Seems like a test on some legacy behaviour?

Reply:
I think it may be, but the events schema now include both.
https://github.com/getsentry/snuba/blob/master/snuba/datasets/events_processor.py#L50-L73

The good thing is this is going away with the events table as soon as we deploy the errors table since it does not have both fields.